### ISSUE TITLE
Version 3.4.0 of the mbedtls crypto library

### ISSUE THREAD

**Issue by darix** [enwi/hueplusplus#88]
**Describe the bug**:

```
hueplusplus-1.0.0/src/EntertainmentMode.cpp:24:10: fatal error: mbedtls/certs.h: No such file or directory
```

Full build log:
https://build.opensuse.org/package/live_build_log/openSUSE:Factory/OpenRGB/standard/x86_64

**Comment by Jojo-1000** [comment]
This is due to a breaking change in version 3 of mbedtls which is already fixed on the development branch.
There is an issue on visual studio when including v3.3.0+ in a cpp context Mbed-TLS/mbedtls#7087 [](https://github.com/Mbed-TLS/mbedtls/issues/7087) (confirmed with VS 2019), so I can only update our version to 3.2.1.

Maybe @enwi can release a 1.1.1 version, as we have updated the mbedtls version and fixed the google test download since last release.

**Comment by darix** [comment]
yeah when we can see a release that fixes the issue would have been my next question. I already opened a bug with OpenRGB to update their intree copy with a reference to this bug.

**Comment by enwi** [comment]
Can this be closed @Jojo-1000 ?

**Comment by darix** [comment]
based from the comment on #93 it isnt fully in yet? :)

**Comment by enwi** [comment]
It should be. I rebased all branches and made sure all commits of the dev and main branches are on each branch

**Comment by darix** [comment]
so release when? :)

**Comment by enwi** [comment]
The code is released on the master branch with version 1.2.0. There is no "GitHub Release" yet.

**Comment by enwi** [comment]
It seems @Jojo-1000 already made an official release https://github.com/enwi/hueplusplus/releases/tag/v1.2.0
